lib/modules/manager/gitlabci/extract.ts

```diff
@@ -120,6 +120,7 @@ function getAllIncludeComponents(
 
 function extractDepFromIncludeComponent(
   includeComponent: GitlabIncludeComponent,
+  registryAliases?: Record<string, string>,
```

Comment:
```suggestion
  registryAliases: Record<string, string> | undefined,
```
I had preferred this

Reply:
I'll keep that in mind for the future... but this has already been merged so what, if anything, should be done now?